__tests__/kata2.booleanToWord.test.js

```diff
@@ -1,5 +1,11 @@
-const { booleanToWord } = require('../src');
+const booleanToWord = require('../src/kata2.booleanToWord');
 
 describe('booleanToWord', () => {
-  // how do we create specs again???
+  it('returns \'Yes\' when passed true',() => {
+    expect(booleanToWord(true)).toEqual(true)
```

Comment:
This should be:

```js
expect(booleanToWord(true)).toEqual('Yes');
```

Equally, your other test should test for a return value of `No`.